public/data/RayanAbid.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "Rayan Abid",
+  "type": "personal",
+  "bio": "Software developer!",
+  "avatar": "https://github.com/rayanabid.png",
+  "links": [
+    {
+      "name": "Checkout my website",
+      "url": "https://www.rayanabid.com/",
+      "icon": "link"
+    },
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/rayanabid",
+      "icon": "github"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/rayanDoesTech",
+      "icon": "twitter"
+    }
+  ],
```

Comment:
Kindly remove this Trailing comma from there! 🛠
```suggestion
  ]
```

Reply:
Thank you. I was wondering what might be causing this.